src/kirby/components/chart/options/activitygauge.ts

```diff
@@ -10,9 +10,11 @@ export const ActivityGaugeOptions: Options = {
     },
     backgroundColor: 'transparent',
     type: 'solidgauge',
-    description: '',
     height: '110%',
   },
+  accessibility: {
+    description: '',
```

Comment:
This looks bad (for accessibility)... won't people with sight disabilities have a problem figuring out what the graph is (based on this option)?